kubernetes/zenko/templates/maintenance/retry-failed-cron.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.maintenance.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "zenko.fullname" . }}-retry-failed
+  labels:
+    app: {{ template "zenko.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
@ssalaues 
We should probably make a template for this:
{{/*
Create chart name and version as used by the chart label.
*/}}
{{- define "zenko.chart" -}}
{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
{{- end -}}